sql/wsrep_mysqld.cc

```diff
@@ -2870,6 +2870,57 @@ static void wsrep_RSU_end(THD *thd) {
   thd->disable_binlog_guard.reset();
 }
 
+bool thd_enter_apply_monitor(THD* thd) {
+
+  if (thd->killed != THD::NOT_KILLED) {
+    return true;
+  }
+  if (thd->slave_thread) {
+    Slave_worker * sw = dynamic_cast<Slave_worker*>(thd->rli_slave);
```

Comment:
Can we use static_cast here? If not, we should check for sw not being nullptr

Reply:
`Slave_worker` is a child class of `Relay_log_info`. So `dynamic_cast` is the better option here.